src/eve/concepts.py

```diff
@@ -269,3 +269,7 @@ class VType(datamodels.FrozenModel):
 
     # Unique name
     name: str
+
+
+def eq_nonlocated(a: Node, b: Node) -> bool:
+    return utils.ddiff(a, b, ignore_type_in_groups=[SourceLocation, SourceLocationGroup])
```

Comment:
probably in the wrong place. suggestions?

Reply:
Since I guess this is only used in tests, why don't you add it as an utility there? Maybe even define it as a `nonlocated_node_comparator` fixture which returns the comparator function and can be easily used by any test function. By the way, don't you need to explicitly build the `bool` return value to make type checker happy?
```suggestion
def eq_nonlocated(a: Node, b: Node) -> bool:
    return bool(utils.ddiff(a, b, ignore_type_in_groups=[SourceLocation, SourceLocationGroup]))
```

Reply:
No, not necessarily only for tests.